lib/backend/k8s/k8s_fv_test.go

```diff
@@ -747,10 +726,17 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 			kvps, err := c.List(model.GlobalBGPPeerListOptions{})
 			Expect(err).ToNot(HaveOccurred())
 			Expect(kvps).To(HaveLen(2))
-			Expect(kvps[0].Key).To(Equal(kvp1b.Key))
-			Expect(kvps[0].Value).To(Equal(kvp1b.Value))
-			Expect(kvps[1].Key).To(Equal(kvp2b.Key))
-			Expect(kvps[01].Value).To(Equal(kvp2b.Value))
+			keys := []model.Key{}
+			vals := []interface{}{}
+			for _, k := range kvps {
```

Comment:
Would be better if we used a map so we can assert that a specific value exists for a given key.